package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pokt-foundation/pocketjs-monorepo",
-  "version": "2.2.1",
+  "version": "2.2.0-rc.5",
```

Comment:
Why are we going from 2.2.1 -> 2.2.0?

Shouldn't it be either 2.3.0?

Reply:
The version in code was 2.2.1, but the latest version published to NPM was 2.2.0-beta.0 and the latest tag of the repo is v2.1.1.  It seems that v2.2.x was never published.  I think v2.2.0 is the next version.